src/components/activity/ActivityRegistrator.tsx

```diff
@@ -85,13 +85,17 @@ export const ActivityRegistrator = ({ route, navigation }: any) => {
     if (toHour === 0) {
       toHour = 24;
     }
+    
+    // Add a 0 infront of numbers lower than 10
+    const zeroPadding = (value: number) => {
+      if(value < 10) return '0' + value
+      return value.toString()
+    };
 
-    const isoDateString = date.getFullYear() + '-' + date.getMonth() + '-' + date.getDate();
+    const isoDateString = date.getFullYear() + '-' + zeroPadding(date.getMonth()+1) + '-' + zeroPadding(date.getDate());
```

Comment:
We have a function in utils which does this now. It might be a good idea to merge master and change it before merging.